### ISSUE TITLE
Crashes Upon Server Join

### ISSUE THREAD

**Issue by R00tB33rMan** [holebois/minion_skin#1]
Error:
```
java.lang.IllegalAccessError: class com.holebois.minion_skin.client tried to access private method 'void net.minecraft.class_2803.<init>(net.minecraft.class_2540)' (com.holebois.minion_skin.client and net.minecraft.class_2803 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @ae45eb6)
    at com.holebois.minion_skin.client.sendInfoPacket(client.java:28)
    at com.holebois.minion_skin.client.lambda$onInitializeClient$0(client.java:18)
    at net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents.lambda$static$2(ClientPlayConnectionEvents.java:50)
    at net.fabricmc.fabric.impl.networking.client.ClientPlayNetworkAddon.onServerReady(ClientPlayNetworkAddon.java:60)
    at net.minecraft.class_634.handler$bjl000$fabric-networking-api-v1$handleServerPlayReady(class_634.java:3028)
    at net.minecraft.class_634.method_11120(class_634.java:474)
    at net.minecraft.class_2678.method_11567(class_2678.java:69)
    at net.minecraft.class_2678.method_11054(class_2678.java:17)
    at net.minecraft.class_2600.method_11072(class_2600.java:25)
    at net.minecraft.class_1255.method_18859(class_1255.java:162)
    at net.minecraft.class_4093.method_18859(class_4093.java:23)
    at net.minecraft.class_1255.method_16075(class_1255.java:136)
    at net.minecraft.class_1255.method_5383(class_1255.java:121)
    at net.minecraft.class_310.method_1523(class_310.java:1275)
    at net.minecraft.class_310.method_1514(class_310.java:888)
    at net.minecraft.client.main.Main.main(Main.java:265)
    at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
    at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
    at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
    at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
    at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
    at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

**Comment by activepass** [comment]
~~What mc version are you running?~~

Tracked down to crash on 20.5/6, will add support.

**Comment by activepass** [comment]
New version for 1.20.6 can be found:
modrinth - https://modrinth.com/mod/client-info-fix/version/1.0.4+1.20.6
releases - https://github.com/holebois/minion_skin/releases/tag/v1.0.4%2B1.20.6

**Comment by R00tB33rMan** [comment]
> New version for 1.20.6 can be found: modrinth - https://modrinth.com/mod/client-info-fix/version/1.0.4+1.20.6 releases - https://github.com/holebois/minion_skin/releases/tag/v1.0.4%2B1.20.6

Perfect. Thank you!
